daemon/algod/api/algod.oas2.json

```
@@ -2295,8 +2295,16 @@
             "format": "byte"
           }
         },
+        "budget-added": {
+          "description": "Budget added during execution of app call transaction.",
+          "type": "integer"
+        },
+        "budget-consumed": {
+          "description": "Budget consumed during execution of app call transaction.",
+          "type": "integer"
+        },
         "cost": {
-          "description": "Execution cost of app call transaction",
+          "description": "Net cost of app execution. Will be DEPRECATED.",
```

@algoidurovic Minor:  A couple of minor deprecation suggestions.  If accepted, friendly reminder to regenerate `daemon/algod/api/algod.oas3.yml`.  Feel welcomed to modify verbiage.

* I think we're saying the field _is_ deprecated rather than _will be_ deprecated.
* Propose mentioning the deprecation implies possible field removal along with a pointer to replacement fields.  

```suggestion
          "description": "Net cost of app execution. Field is DEPRECATED and is subject for removal.  Instead, use `budget-added` and `budget-consumed`.",
```